### PR TITLE
[#14] [UI] As a user, when I want to search google with a list of keywords, I can upload a keywords file

### PR DESCRIPTION
**Pull request by suho**
#14

## What happened 👀

In order to support the user to search google with a list of keywords, we need to support upload keywords file features
 
## Insight 📝

The home screen includes:
- The upload file button
- Download template file button
 
## Proof Of Work 📹

Screen Shot 2021-11-23 at 16 10 39

### CODE REVIEW

File: app/assets/stylesheets/keywords.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Keywords controller here.
```
Comment (junan) [inline app/assets/stylesheets/keywords.scss]: How about removing this auto-generated file now as we are not using it also it is not aligning with our [CSS structure convention](https://nimblehq.co/compass/development/code-conventions/css/#stylesheets-structure).

File: app/helpers/keywords_helper.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module KeywordsHelper
```
Comment (junan) [inline app/helpers/keywords_helper.rb]: How about removing this auto-generated file also as we are not using it? If needed we can add it later.

File: spec/requests/keywords_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe KeywordsController, type: :request do
+  describe 'GET #index' do
+    it 'returns http success' do
+      sign_in Fabricate(:user)
+      get :index
+      expect(response).to have_http_status(:success)
```
Comment (junan) [inline spec/requests/keywords_controller_spec.rb:11]: We usually keep the assert matching stuff to a separate line as a group. To make it more readable the tests
```suggestion

      expect(response).to have_http_status(:success)
```

File: app/assets/stylesheets/keywords.scss (hunk 1 above)
Comment (suho) [reply app/assets/stylesheets/keywords.scss]: Thank you! Updated in [`067c022` (#37)](https://github.com/suho/google-search-ruby/pull/37/commits/067c0225bed0a50f9947e8747b64a6b210c6e73d)

File: app/helpers/keywords_helper.rb (hunk 2 above)
Comment (suho) [reply app/helpers/keywords_helper.rb]: Good catch! Updated in [`067c022` (#37)](https://github.com/suho/google-search-ruby/pull/37/commits/067c0225bed0a50f9947e8747b64a6b210c6e73d)

File: spec/requests/keywords_controller_spec.rb (hunk 3 above)
Comment (suho) [reply spec/requests/keywords_controller_spec.rb:11]: Updated in [`067c022` (#37)](https://github.com/suho/google-search-ruby/pull/37/commits/067c0225bed0a50f9947e8747b64a6b210c6e73d)

**Review by junan — APPROVED** [review]

**Review by hanam1ni — APPROVED** [review]

**Pull request merged**